activestorage/lib/active_storage.rb

```diff
@@ -55,6 +55,26 @@ module ActiveStorage
   mattr_accessor :service_urls_expire_in, default: 5.minutes
   mattr_accessor :routes_prefix, default: "/rails/active_storage"
 
+  mattr_accessor :blob_class, default: "ActiveStorage::Blob"
```

Comment:
Looking at [how we do this in other parts of Rails](https://github.com/rails/rails/blob/6a50498618c2b9a316c3b28f6cb45bdaa70726ad/actionmailer/lib/action_mailer/railtie.rb#L49-L51), there is no general load hook for `:active_storage`, or more specifically, no central piece (eg. base class) to trigger load from. We have tests against autoloading during initialization, so I've deferred loading of model classes by reference.

There is probably a more straightforward way of doing this.